test/extrapolation/runtests.jl

```diff
@@ -27,6 +27,8 @@ etpf = @inferred(extrapolate(itpg, NaN))
 @test_throws BoundsError etpf[2.5,2]
 @test_throws ErrorException etpf[2.5,2,1]  # this will probably become a BoundsError someday
 
+@inferred(getindex(etpf, dual(-2.5,1)))
+
 etpf = @inferred(FilledExtrapolation(itpg, 'x'))
```

Comment:
This test can probably just be dropped, since it was really testing explicit control.